Microsoft.PowerShell_profile.ps1

```diff
@@ -9,13 +9,30 @@ function prompt {
 $Host.PrivateData.DebugBackgroundColor = "DarkGray"
 $Host.PrivateData.ErrorBackgroundColor = "DarkGray"
 
-Set-PSReadlineOption -TokenKind Command   -ForegroundColor White
-Set-PSReadlineOption -TokenKind Comment   -ForegroundColor DarkGreen
-Set-PSReadlineOption -TokenKind Keyword   -ForegroundColor Blue
-Set-PSReadlineOption -TokenKind Member    -ForegroundColor Cyan
-Set-PSReadlineOption -TokenKind Number    -ForegroundColor Yellow
-Set-PSReadlineOption -TokenKind Operator  -ForegroundColor Magenta
-Set-PSReadlineOption -TokenKind Parameter -ForegroundColor Cyan
-Set-PSReadlineOption -TokenKind String    -ForegroundColor Yellow
-Set-PSReadlineOption -TokenKind Type      -ForegroundColor Green
-Set-PSReadlineOption -TokenKind Variable  -ForegroundColor White
+
+$promptColors = @{
+    'Command' = 'White'
+    'Comment' = 'DarkGreen'
+    'Keyword' = 'Blue'
+    'Member' = 'Cyan'
+    'Number' = 'Yellow'
+    'Operator' = 'Magenta'
+    'Parameter' = 'Cyan'
+    'String' = 'Yellow'
+    'Type' = 'Green'
+    'Variable' = 'White'
+}
+if((Get-Module -Name "PSReadline").Version -gt 1.0.0) {
```

Comment:
It might be better to only check the major version. The [lzybkr version ](https://github.com/lzybkr/PSReadLine/blob/master/PSReadLine/AssemblyInfo.cs) has major version as 2.
```suggestion
if((Get-Module -Name "PSReadline").Version.Major -eq 2) {
```